Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
```

Comment:
I believe it would be more powerful and useful project if we have an unified crate, dep and generator and easy to use apis. Users just need to clone the project, run generator and add a-g-a to their dependencies. If we need to publish the crate, it should be done with the tested generated codes or it's useless (that this is not the case we're talking about.)

And answer to your question,  yes, it will create a new crate repository as its name called `async-google-apis`.